ffmpeg (FFmpeg): update to 7.0.2

MingcongBai opened the pull request:
Topic Description
-----------------

Update FFmpeg to 7.0.2 and rebuild reverse dependencies as needed.

Package(s) Affected
-------------------

- aegisub: 3.2.2+git20231203-2
- akode: 14.1.2-2
- alsa-plugins: 1.2.12-1
- aubio: 0.4.9-4
- audacious: 4.4-4
- blender: 4.0.2-3
- chromaprint: 1.5.1-1
- cmus: 2.11.0-2
- corsixth: 1:0.67-1
- ddnet: 18.4-1
- deadbeef: 1.9.6-2
- digikam: 8.4.0-2
- dolphin-emu: 1:2407-2
- ffmpeg: 7.0.2
- ffmpegthumbnailer: 2.2.2-1
- ffmpegthumbs: 23.08.5-1
- ffms2: 5.0
- freerdp: 2:3.6.3
- gegl-0.4: 0.4.48-2
- gnome-boxes: 42.3-2
- goldendict: 1:1.5.0-1
- gpac: 2.4.0
- gstreamer: 1.24.7
- gtk-4: 4.14.2-1
- gtk-update-icon-cache: 4.14.2-1
- guvcview: 2.1.0-1
- haruna: 0.12.3-2
- harvid: 0.9.1
- k3b: 23.08.5-2
- k3b-trinity: 14.1.2-2
- k9copy-trinity: 14.1.2-1
- kfilemetadata: 5.115.0-1
- kid3: 3.9.5
- kodi: 1:21.1
- kpipewire: 5.27.11-1
- libde265: 1.0.8-4
- makemkv: 1.17.7
- mgba: 0.10.3-1
- mixxx: 2.4.1-4
- mlt: 7.26.0
- mocp: 2.6.0~svn.r3005
- moonlight-embedded: 2.5.3-2
- moonlight-qt: 4.3.0-2
- mpd: 0.23.15-6
- mplayer: 1:1.5
- mpv: 0.38.0-1
- neatvnc: 0.7.1-1
- obs-studio: 30.2.3-1
- openmw: 0.48.0+git20240902
- openscenegraph: 3:3.6.5-4
- osu-lazer: 2024.817.0-1
- owntone: 28.9-2
- pipewire: 1.2.3-1
- ppsspp: 1.17.1-1
- qmmp: 1.6.5-2
- qt-5: 1:5.15.13+webengine5.15.16+webkit5.212.0+kde20240408-6
- qt-5-doc: 1:5.15.13+webengine5.15.16+webkit5.212.0+kde20240408-6
- qt-6: 6.7.2-3
- remmina: 1.4.35
- scrcpy: 2.4-1
- simplescreenrecorder: 0.4.4
- telegram-desktop: 5.4.1
- tigervnc: 1.14.0+xserver21.1.13
- tracker-miners: 3.3.1-3
- unpaper: 1:7.0.0
- vlc: 3.0.21-3
- weston: 13.0.3
- xine-lib: 1.2.13-2
- xjadeo: 0.8.13-1
- xpra: 6.1.2

Security Update?
----------------

No

Build Order
-----------

```
#buildit ffmpeg:-pkgbreak groups/ffmpeg-rebuilds groups/ffms2-rebuilds groups/freerdp-rebuilds tigervnc ffmpeg
```

Test Build(s) Done
------------------

**Primary Architectures**

- [x] AMD64 `amd64`
- [x] AArch64 `arm64`
- [x] LoongArch 64-bit `loongarch64`

**Secondary Architectures**

- [x] Loongson 3 `loongson3`
- [x] PowerPC 64-bit (Little Endian) `ppc64el`
- [x] RISC-V 64-bit `riscv64`

aosc-buildit-bot committed:
freerdp: update to 3.6.3

- Add fuse-3, sdl2-ttf, webkit2gtk deps.
- Drop an upstreamed patch.
- Add PKGBREAK due to SONAME changes

aosc-buildit-bot committed:
mplayer: update to 1.5

- (Loong Arch Linux) Add a patch to fix loongarch64 build.
- Backport (a fuck ton) of upstream patches to fix build with FFmpeg 7.0.
- Track patches at AOSC-Tracking/mplayer @ aosc/v1.5.
- Drop vendored FFmpeg source... seriously, it causes a lot of pain (MPlayer
  uses ambiguous "" references to headers, causing system and vendor headers
  to mangle).
- Drop unused retro <= * symlinks.

aosc-buildit-bot committed:
tigervnc: update to 1.14.0

- Add ffmpeg dep for H.264 codec support.
- Clean up spec and use UPSTREAM_VER (as well as removing a hard-coded v1.12.0
  version reference!).
- Switch to xorg-server 21.x per upstream requirement.
- Add libxcvt dep.

Co-authored-by: 白铭骢 (Mingcong Bai) (@MingcongBai) <jeffbai@aosc.io>

MingcongBai committed:
tinyxml2: new, 10.0.0

MingcongBai committed:
ffmpeg: update to 7.0.2

- Clean up build script and incorporate Afterglow recipes.
- Update Loongson (MIPS) patches for 7.0.2.
- Update PKGBREAK.
- Drop unused --enable-avresample option.
- Clean up references to mips64r6el.
- (Fedora patch) add av_stream_get_first_dts() for Chromium
- (loongarch64) Disable LTO.

FIXME: lto1: fatal error: target specific builtin not available
Potentially a compiler bug?

See also a similar (resolved) bug for ppc64el.[^1]

[^1]: https://gcc.gnu.org/bugzilla/show_bug.cgi?id=102347

Co-authored-by: Henry Chen <chenx97@aosc.io>

MingcongBai committed:
groups: add ffmpeg-rebuilds

MingcongBai committed:
blender: rebuild for FFmpeg 7.0

Track patches at AOSC-Tracking/ffmpeg @ aosc/v4.0.2.

MingcongBai committed:
mixxx: rebuild for FFmpeg 7.0

MingcongBai committed:
guvcview: rebuild for FFmpeg 7.0

MingcongBai committed:
scrcpy: rebuild for FFmpeg 7.0

MingcongBai committed:
unpaper: update to 7.0.0

- Fix compatibility with FFmpeg >= 5.0.
- Replace DocBook with Sphinx.

MingcongBai committed:
citra: drop, gone

MingcongBai committed:
corsixth: rebuild for FFmpeg 7.0

MingcongBai committed:
ddnet: rebuild for FFmpeg 7.0

MingcongBai committed:
dolphin-emu: rebuild for FFmpeg 7.0

MingcongBai committed:
mgba: rebuild for FFmpeg 7.0

MingcongBai committed:
osu-lazer: rebuild for FFmpeg 7.0

MingcongBai committed:
ppsspp: use bundled FFmpeg

- Mark loongson3 as FAIL_ARCH.
- Remove special GL preferences for AArch64.

MingcongBai committed:
stepmania: drop, orphaned

MingcongBai committed:
goldendict: rebuild for FFmpeg 7.0

Import FFmpeg 7.0 patch from AOSC-Tracking.

MingcongBai committed:
aubio: rebuild for FFmpeg 7.0

- Backport FFmpeg 7.0 patches (AOSC-Tracking/ffmpeg @ aosc/0.4.9).
- Disable tests due to lack of upstream test maintenance.

MingcongBai committed:
cmus: rebuild for FFmpeg 7.0

MingcongBai committed:
deadbeef: rebuild for FFmpeg

Backport upstream fix for FFmpeg 7.0 (AOSC-Tracking/deadbeef @ aosc/1.9.6).

MingcongBai committed:
ffmpegthumbnailer: rebuild for FFmpeg 7.0

Backport upstream patches for FFmpeg >= 5.0
(AOSC-Tracking/ffmpegthumbnailer @ aosc/2.2.2).

MingcongBai committed:
ffms2: update to 5.0

MingcongBai committed:
groups: add ffms2-rebuilds

MingcongBai committed:
aegisub: rebuild for ffms2 5.0

MingcongBai committed:
harvid: update to 0.9.1

- Drop an upstreamed patch.
- This version ships with support for FFmpeg 7.0.

MingcongBai committed:
kid3: update to 3.9.5

- This version ships with support for FFmpeg 7.0.
- Ensure that the package is built with Qt 5 as we do not yet ship KDE 6.
- Use cmakeninja ABTYPE.

MingcongBai committed:
makemkv: update to 1.17.7

This version comes with support for FFmpeg 7.0.

MingcongBai committed:
mocp: (Debian, AOSC patches) update to 2.6.0~svn.r3005

- Sync patchset from Debian since this package lost upstream maintenance.
- Fix build on FFmpeg 7.0 (AOSC-Tracking/mocp @ aosc/2.6.0-svn-r3005).

MingcongBai committed:
moonlight-embedded: rebuild for FFmpeg 7.0

MingcongBai committed:
moonlight-qt: rebuild for FFmpeg 7.0

MingcongBai committed:
chromaprint: (Debian patches) rebuild for FFmpeg 7.0

Sync Debian FFmpeg >= 5.0 patches (AOSC-Tracking/chromaprint @ aosc/v1.5.1).

MingcongBai committed:
mpd: rebuild for FFmpeg 7.0

MingcongBai committed:
mpv: rebuild for FFmpeg 7.0

MingcongBai committed:
obs-studio: rebuild for FFmpeg 7.0

MingcongBai committed:
owntone: rebuild for FFmpeg 7.0

MingcongBai committed:
pipewire: rebuild for FFmpeg 7.0

MingcongBai committed:
qmmp: rebuild for FFmpeg 7.0

MingcongBai committed:
simplescreenrecorder: update to 0.4.4 and fix build with FFmpeg 7.0

- Backport upstream fixes for FFmpeg >= 7.0
  (AOSC-Tracking/simplescreenrecorder @ aosc/0.4.4).
- Merge elfhacks.h patches for loongarch64, riscv64.

MingcongBai committed:
transcode: drop, orphaned

MingcongBai committed:
vlc: use vendored FFmpeg

With FFmpeg >= 5.0, the libavcodec => libva interface is no longer available.
Using newer FFmpeg would cause higher CPU usage unless the user specifies
VDPAU as the video decoding backend.

Use bundled FFmpeg (4.4) for now.

Ref: https://www.debian.org/releases/stable/amd64/release-notes/ch-information.en.html#vlc-ffmpeg-5

MingcongBai committed:
xjadeo: rebuild for FFmpeg 7.0

MingcongBai committed:
groups: add freerdp-rebuilds

MingcongBai committed:
vinagre: rebuild for FreeRDP 3.6

MingcongBai committed:
weston: rebuild for FreeRDP 3.6

MingcongBai committed:
gnome-boxes: rebuild for FreeRDP 3.6

MingcongBai committed:
remmina: update to 1.4.35

- Rebuild for FreeRDP 3.6.
- Enable NEON optimization for ARM64.

MingcongBai committed:
gtk-frdp: rebuild for FreeRDP 3.6

MingcongBai committed:
gnome-remote-desktop: rebuild for FreeRDP 3.6

MingcongBai committed:
gnome-remote-desktop: drop, orphaned

MingcongBai committed:
gnome-boxes: disable RDP feature due to incompatibility with FreeRDP 3.x

MingcongBai committed:
gtk-frdp: drop, orphaned

MingcongBai committed:
gnome-connections: drop, orphaned

To be re-introduced with a GNOME update, it currently blocks FFmpeg update.

MingcongBai committed:
vinagre: drop, orphaned

This package lost upstream maintenance.

MingcongBai committed:
xpra: update to 6.1.2

- Copy .git metadata for revision report in the about dialogue.
- This release comes with support for FFmpeg 7.0.

MingcongBai committed:
qtox: drop, orphaned

Upstream has ceased maintenance for this project.

MingcongBai committed:
ada: new, 2.9.1

MingcongBai committed:
telegram-desktop: update to 5.5.3

Backport an upstream patch in tg_owt for FFmpeg >= 7.0.

Tracking repositories:

- AOSC-Tracking/telegram-desktop @ aosc/v5.5.3.
- AOSC-Tracking/tg_owt @ aosc/master.

Co-authored-by: Kaiyang Wu <origincode@aosc.io>
Signed-off-by: Kaiyang Wu <origincode@aosc.io>

MingcongBai committed:
gtk-4: rebuild for FFmpeg 7.0

MingcongBai committed:
tracker-miners: switch to GStreamer media extractor

Upstream never adapted its libav extractor to FFmpeg >= 5.0.

MingcongBai committed:
ffmpegthumbs: rebuild for FFmpeg 7.0

MingcongBai committed:
haruna: rebuild for FFmpeg 7.0

MingcongBai committed:
qt-5: rebuild for FFmpeg

- (Arch Linux patch) Fix build with FFmpeg >= 5.0.
- (openSUSE patch) Fix build with FFmpeg >= 7.0.
- (Upstream revert) Allow building with FFmpeg >= 5.0.

MingcongBai committed:
k3b: rebuild for FFmpeg 7.0

Backport a patch for FFmpeg >= 7.0 (AOSC-Tracking/k3b @ aosc/v23.08.5).

MingcongBai committed:
digikam: rebuild for FFmpeg 7.0

Drop qtav dep, as upstream removed QTAV bindings in favour of their own
libqtavplayer plugin. [^1]

[^1]: Since 6293eec17129051f826a104ae9288eeda7495091.

MingcongBai committed:
kfilemetadata: rebuild for FFmpeg 7.0

MingcongBai committed:
kpipewire: rebuild for FFmpeg 7.0

MingcongBai committed:
akode: rebuild for FFmpeg 7.0

Track patches at AOSC-Tracking/akode @ aosc/r14.1.2.

MingcongBai committed:
k3b-trinity: rebuild for FFmpeg 7.0

Track patches at AOSC-Tracking/ffmpeg @ aosc/r14.1.2.

MingcongBai committed:
k9copy: rebuild for FFmpeg 7.0

MingcongBai committed:
qt-6: rebuild for FFmpeg 7.0

- Backport an upstream patch to fix build with FFmpeg >= 6.1.
- Track patches at AOSC-Tracking/qt-6 @ aosc/v6.7.2.

MingcongBai committed:
qtav: drop, orphaned

MingcongBai committed:
gegl-0.4: rebuild for FFmpeg 7.0

- Backport an upstream fix for FFmpeg >= 7.0.
- Track patches at AOSC-Tracking/gegl @ aosc/GEGL_0_4_48.

MingcongBai committed:
gstreamer: update to 1.24.7

- This version was released with FFmpeg 7.0 support.
- Disable broken Rust bindings.

MingcongBai committed:
libde265: rebuild for FFmpeg 7.0

MingcongBai committed:
mlt: update to 7.26.0

This version was released with FFmpeg 7.0 support.

MingcongBai committed:
mlt-6: drop, orphaned

MingcongBai committed:
synfig: change dep mlt-6 => mlt

- Adapt build scripts for Autobuild4.
- Use Git tag instead of generated tarballs.

MingcongBai committed:
xine-lib: rebuild for FFmpeg 7.0

- Backport upstream patches for FFmpeg 7.0 support.
- Track patches at AOSC-Tracking/xine-lib @ aosc/v1.2.13.

MingcongBai committed:
neatvnc: rebuild for FFmpeg 7.0

MingcongBai committed:
openmw: update to latest snapshot (20240902)

- Move unshield to run-time dependency.
- Improve PKGDES.
- Sync FAIL_ARCH from luajit.

FIXME: 0.48.0 does not come with FFmpeg >= 5.0 support and upstream patch
is next to impossible to backport.

MingcongBai committed:
kodi: update to 21.1

- (Debian) Backport patches for FFmpeg 7.0 support.
- Fix build with RelWithDebInfo CMake build type, as libcrossguid finds the
  incorrect library name with this "kind of debug" build type.
- Add tinyxml2 dep.
- Install Copacetic Extra Fonts as required.
- Track patches at AOSC-Tracking/kodi @ aosc/21.1-Omega.

MingcongBai committed:
audacious: rebuild for FFmpeg 7.0

MingcongBai committed:
openscenegraph: disable FFmpeg plugin

Upstream has not been keeping up with FFmpeg updates and has not seen active
feature development for a while. Drop FFmpeg to save our own time (we still
have GStreamer).

Debian dropped dependency to FFmpeg, disabling this plugin implicitly. Fedora,
on the other hand, never enabled this plugin.

Do this more explicitly so we don't run into trouble if the build environment
came with FFmpeg installed.

MingcongBai commented:
Building manually to avert unstable Internet connections on some buildbots.

MingcongBai committed:
gnome-control-center: drop unused vinagre dep

OriginCode commented:
Smoke tests on amd64:
blender
![圖片](https://github.com/user-attachments/assets/bcf94613-9f3e-4c98-a52a-887501377002)
mixxx
![圖片](https://github.com/user-attachments/assets/2c06d372-ceb6-4e23-85e5-869516812e19)
guvcview
![圖片](https://github.com/user-attachments/assets/9689a7e2-467e-43d8-9e30-c535ba087c46)
scrcpy
![圖片](https://github.com/user-attachments/assets/ba4d7e0f-e14a-4107-aabe-c22ed33d8a24)
unpaper (I don't have scanned files)
![圖片](https://github.com/user-attachments/assets/5cfb5bfc-e1d0-4888-9a41-b2311281bf31)
corsixth
![圖片](https://github.com/user-attachments/assets/68bb2fc6-b49c-4da1-a7c0-f1baad9a32b4)
ddnet
![圖片](https://github.com/user-attachments/assets/02f4382f-f976-41fe-900b-f8b5ecd14ce9)
dolphin-emu
![圖片](https://github.com/user-attachments/assets/198c43c4-3354-469b-baa3-23aef02fc006)
mgba
![圖片](https://github.com/user-attachments/assets/20d1a9db-4ced-42fc-82c5-13167fa383f6)
osu-lazer
![圖片](https://github.com/user-attachments/assets/b5afd174-21ed-4d9b-b0a0-2e80f6911f35)
ppsspp
![圖片](https://github.com/user-attachments/assets/d05d00eb-65b2-4c6c-824f-e5993c37c83d)
goldendict
![圖片](https://github.com/user-attachments/assets/a9e490e7-98b1-46d3-808a-c0c38f87d886)
aubio (No valid test files)
![圖片](https://github.com/user-attachments/assets/b911307f-9327-4617-926d-d9151734fbd7)
cmus
![圖片](https://github.com/user-attachments/assets/49c1916c-81f3-4f67-8097-3ecb9f415d11)
deadbeef
![圖片](https://github.com/user-attachments/assets/748ea939-ad2b-4bdb-8009-81ef19821e1d)
ffmpegthumbnailer
![圖片](https://github.com/user-attachments/assets/1dc568c5-72f5-40d2-9247-e6b9fb0f5443)
aegisub **SIGSEGV**
![圖片](https://github.com/user-attachments/assets/0d78e4aa-d5a2-439f-9191-33bd5096c139)
gpac **Black screen on amd64, SIGSEGV on loongarch64**
harvid (No test env)
![圖片](https://github.com/user-attachments/assets/bb994441-dbdd-48d0-8b5a-e037cce472c6)
kid3
![圖片](https://github.com/user-attachments/assets/590bd583-454c-478c-ae4b-d3e6de8ac8e9)

OriginCode commented:
makemkv (**PENDING DROP, ORPHANING**)
![圖片](https://github.com/user-attachments/assets/598669f5-d5d9-430d-abe1-137f9d7eff59)
mocp
![圖片](https://github.com/user-attachments/assets/b6a68c3c-3a81-4e6f-a225-19b89354d746)
moonlight-embedded
![圖片](https://github.com/user-attachments/assets/af5acde2-90ec-49f8-8877-66cf297618dc)
moonlight-qt
![圖片](https://github.com/user-attachments/assets/9ae7776f-3370-4ab0-9d6a-797c4c38b994)
mpd
![圖片](https://github.com/user-attachments/assets/a0e9afb4-392b-4cae-997d-259d17dd7e39)
mpv
![圖片](https://github.com/user-attachments/assets/6407a855-b1bc-4646-bb3c-f64cebae6fb6)
obs-studio
![圖片](https://github.com/user-attachments/assets/1a57f19a-8eb7-4c44-b38f-4172a3442c4c)
owntone
![圖片](https://github.com/user-attachments/assets/6a95cfaf-233e-4903-8f72-98c6d25008b8)
pipewire: working
qmmp
![圖片](https://github.com/user-attachments/assets/c0c6af72-e7a9-4a54-8833-ecf5262b8780)

OriginCode commented:
simplescreenrecorder, haruna
![Screenshot_20240919_015857](https://github.com/user-attachments/assets/3a8b640b-3441-4da0-a558-327b19e915ab)

OriginCode commented:
vlc
![圖片](https://github.com/user-attachments/assets/0c74fa2f-844f-433d-a656-7d87eb660eb8)
xjadeo
![圖片](https://github.com/user-attachments/assets/891cc695-3471-40b0-b06b-64412cd62d4a)
freerdp
![圖片](https://github.com/user-attachments/assets/1dcb28a1-6246-4416-bf21-c114ea4e3170)
remmina
![圖片](https://github.com/user-attachments/assets/4af97e35-a59f-4d62-b1ca-eea796e14d9d)
gnome-boxes (**SIGTRAP, conflicting symbols**)
![圖片](https://github.com/user-attachments/assets/dfa4a523-695b-411d-a819-26c1fdf1a97a)
xpra (**Need copy-repo**)
![圖片](https://github.com/user-attachments/assets/e08af922-1fa7-40d3-97df-02fb410f00af)
telegram-desktop
![圖片](https://github.com/user-attachments/assets/13f85f5f-6598-43ac-81e9-c60096170a1d)
tracker-miners (**No test env**)

OriginCode commented:
k3b
![圖片](https://github.com/user-attachments/assets/37c847d6-58bb-43b6-abb7-80930021ef2c)
digikam
![圖片](https://github.com/user-attachments/assets/51194649-6eb5-4e40-b2c6-c8479cddcb84)
k3b-trinity
![圖片](https://github.com/user-attachments/assets/ff38d9b4-00cc-4509-8229-40572da290f3)
k9copy-trinity
![圖片](https://github.com/user-attachments/assets/90517dcc-1b53-4b55-9073-cfafed8fc31a)
synfig (**Missing reworked package**)
openmw (**Installs 3rd party software, and fails to do so**)
![圖片](https://github.com/user-attachments/assets/e4670b57-0ee4-4e63-a179-f6e426dba1c6)
![圖片](https://github.com/user-attachments/assets/6306e8e4-341e-47a9-b27d-99fd0c6a6205)
kodi (**Black screen when playing video, audio works**)
![圖片](https://github.com/user-attachments/assets/e614c0c7-c9bd-4aab-8171-e46ceefd0e05)
audacious
![圖片](https://github.com/user-attachments/assets/24e8d881-8e3a-4e16-b06a-0f518b734856)
mplayer
![圖片](https://github.com/user-attachments/assets/173e22d5-7400-4175-9954-09e2bd20371b)
hmcl
![圖片](https://github.com/user-attachments/assets/576d48ea-7c71-477a-add9-216d7b2adb7c)
openscenegraph
![圖片](https://github.com/user-attachments/assets/ea99dcf9-e58e-4479-9bbb-c6a367088000)
tigervnc (**Missing font?**)
![圖片](https://github.com/user-attachments/assets/805d33f6-72af-4f2c-b750-f50813d14c3b)

MingcongBai commented:
>  tigervnc (Missing font?)

@OriginCode `vncviewer` is fine here:

![图片](https://github.com/user-attachments/assets/8d8e7cbe-81d0-424e-9a24-473e1df33e00)

MingcongBai committed:
alsa-plugins: rebuild for FFmpeg 7.0

MingcongBai committed:
gnome-boxes: drop, orphaned

This package is difficult to keep maintenance without a GNOME update.

To re-introduce with a future GNOME update.

MingcongBai committed:
makemkv: drop, orphaned

Latest upstream version broke shareware evaluation and never released an
update to address its oversight:

WARNING!!! The latest available version 1.17.7 currently refuses to start
evaluation period and claims that it is too old. Please, do not purchase the
full registration key if you never had a chance to test the program. Instead
please use the current beta key to unlock all features:

 T-WaI15n3jLrzWU88y7liOljd7UIR0mormcZo2oO5xt_9CSj3Mg6mOxR1eUvU_CA569e

Ref: https://www.makemkv.com/buy/ (as of September 24, 2024)

MingcongBai commented:
Fixed OpenMW:

![图片](https://github.com/user-attachments/assets/33d344e1-d3c9-4385-9100-b55d654f8e08)

`gnome-boxes` and `makemkv` has been dropped.

MingcongBai commented:
@OriginCode KODI works well for me here (Wayland issue?):

![图片](https://github.com/user-attachments/assets/a98cbd47-848e-4dbd-a20d-09fd85c9cb01)

MingcongBai commented:
Neither `aegisub` nor `gpac` segfault'd on AMD64, but the latter did segfault on LoongArch. Hmm.

OriginCode commented:
> Neither `aegisub` nor `gpac` segfault'd on AMD64, but the latter did segfault on LoongArch. Hmm.

`aegisub` does not work with native Wayland, upstream issue https://github.com/Aegisub/Aegisub/issues/180
Use `GDK_BACKEND=x11` to force it use X11 as a workaround.

OriginCode commented:
> @OriginCode KODI works well for me here (Wayland issue?):
> 
> ![图片](https://private-user-images.githubusercontent.com/5006263/370303944-a98cbd47-848e-4dbd-a20d-09fd85c9cb01.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjc1MTc1OTUsIm5iZiI6MTcyNzUxNzI5NSwicGF0aCI6Ii81MDA2MjYzLzM3MDMwMzk0NC1hOThjYmQ0Ny04NDhlLTRkYmQtYTIwZC0wOWZkODVjOWNiMDEucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDkyOCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA5MjhUMDk1NDU1WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9Zjc0YTY0MDE5MWE0NjRkYzhiYWE0MWRlMjNkZGRjNTg3MjYxY2MzMjIzY2U0ODQwNzgwMzRkMDIxNDAzMDI0ZCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.2dniotXgsUeZpxhWYcjvKU_OODGxXvTZQgjAfSHO0eM)

Workaround supplying `--viewing=X11` to force it using X11 on Wayland works.

MingcongBai committed:
gpac: drop, orphaned

MingcongBai committed:
ogmrip: drop, orphaned

MingcongBai commented:
Dropping gpac (and ogmrip as a result), the latter is barely maintained and is the sole dependee.

This topic should be good to go. cc @OriginCode

OriginCode commented on app-multimedia/ffmpegthumbnailer/autobuild/defines:
```diff
@@ -3,7 +3,6 @@ PKGSEC=utils
 PKGDEP="gvfs ffmpeg libjpeg-turbo libpng"
 PKGDES="Lightweight video thumbnailer that can be used by file managers"
 
-ABTYPE=cmake
```
Why removing `ABTYPE`?

MingcongBai replied:
Fixed. I meant to specify `cmakeninja`.

jiegec reviewed (commented):
gnome-remote-desktop needs to be removed from gnome-base as well.
